libs/cmap/cmap.go

```diff
@@ -22,31 +22,28 @@ func (cm *CMap) Set(key string, value interface{}) {
 	cm.l.Unlock()
 }
 
-// GetOrSet returns the existing value if present. Othewise, it stores `newValue` and returns it.
-func (cm *CMap) GetOrSet(key string, newValue interface{}) (value interface{}, alreadyExists bool) {
-
+// GetOrSet returns the existing value if present. Otherwise, it stores `newValue` and returns it.
+func (cm *CMap) GetOrSet(key string, newValue interface{}) (value interface{}, ok bool) {
```

Comment:
this `ok` is not clear - it's not ok, it's indicator if it was Get or Set. In [sync.Map](https://pkg.go.dev/sync#Map.LoadOrStore), it is named `loaded`.

```suggestion
func (cm *CMap) GetOrSet(key string, newValue interface{}) (value interface{}, loaded bool) {
```

Reply:
regarding `ok`: the idea is to have a similar approach to how to name the result of key existence into a map. 
there are no strict rules what the name but usually, it is `ok`[e.g.](https://golang.org/doc/effective_go#maps)
I also don't mind about `loaded` as well.
regarding the behaviour of the method `GetOrSet `: the second argument must indicate us only about the existence of the key in a map, it should not be related to the indicator of `Set` operation, 'cause if we are not able to modify the value by a key then the app must panic (or we must support error argument for this use-case)